packages/amplify-category-auth/tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "src/__tests__",
   ],
   "references": [
+    {"path": "../amplify-cli-core"},
     {"path": "../amplify-headless-interface"},
-    {"path": "../amplify-cli-core"}
+    {"path": "../amplify-util-headless-input"},
```

Comment:
it is since this PR without these changes are not doing incremental compilation, the same change meanwhile added by @edwardfoyle after I discovered it.